modules/datastore/src/DatastoreService.php

```diff
@@ -217,8 +217,8 @@ public function getImportService(DataResource $resource): ImportService {
   /**
    * Returns the Data Dictionary fields.
    */
-  public function getDataDictionaryFields() {
-    return $this->dictionaryEnforcer->returnDataDictionaryFields();
+  public function getDataDictionaryFields($id = NULL) {
```

Comment:
Let's document this method properly with docbloc and type/return hints

Reply:
Good call thanks